### PR TITLE
feat: signup form added

### PR DESCRIPTION
**Pull request by som749898**
Please review my component
Screenshot (89)

### CODE REVIEW

File: Signup/signup.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+.signup-heading {
+  text-align: center;
+  padding-bottom: 2.5rem;
```
Comment (Kavita539) [inline Signup/signup.css:4]: suggestion : For spacings you can use variables for better consistency.

**Review by Kavita539 — APPROVED** [review]
Overall looks good

**Review by someshtiwarii — APPROVED** [review]
Good to merge

**Review by koushikrp — APPROVED** [review]
- Good File and Folder Structure
- Good Naming Convention for classes
- Try to separate text-align: center; and remove .signup-text completely
- Good Indentation

File: Signup/signup.css (hunk 1 above)
Comment (som749898) [reply Signup/signup.css:4]: I have used variable for 10px,20px and 30px . For larger size i specify it in a class

Comment (Kavita539) [reply Signup/signup.css:4]: > I have used variable for 10px,20px and 30px . For larger size i specify it in a class

ok I have approved it!

**Comment by prateek1361** [comment]
good to merge

**Pull request merged**